src/sentry/api/endpoints/project_team_details.py

```diff
@@ -37,10 +37,8 @@ def post(self, request: Request, project, team_slug) -> Response:
             team = Team.objects.get(organization_id=project.organization_id, slug=team_slug)
         except Team.DoesNotExist:
             raise Http404
-        if not request.access.has_team_scope(team, "project:write"):
-            return Response(
-                {"detail": ["You do not have permission to perform this action."]}, status=403
-            )
+
+        # A user with project:write can grant access to this project to other user/teams
```

Comment:
is team scope `project:write` for projects within the team? so this is removing it because the team admin doesn't necessarily have to be a member of the team that the project is being linked to?

Reply:
Yes, correct.